webpack/assets/javascripts/react_app/components/toastNotifications/toastListitem/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import helpers from '../../../common/helpers';
+import Timer from '../../../common/Timer';
+import Alert from '../../common/Alert';
+
+import { defaultTimerDelay } from './Toast.consts';
+import './Toast.scss';
+class Toast extends React.Component {
+  constructor(props) {
+    super(props);
+    helpers.bindMethods(this, ['onMouseEnter', 'onMouseLeave']);
+  }
+
+  componentDidMount() {
+    const { sticky, dismiss, timerDelay } = this.props;
+
+    if (!sticky) {
+      this.timer = new Timer(dismiss, timerDelay || defaultTimerDelay);
```

Comment:
I dont think so, since other components may want to use the timer with different default delays